README.md

```diff
@@ -1,6 +1,6 @@
 # sails-forth
 
-Sails-forth is a clojure salesforce library. It is specified with clojure.spec.
+Sails-forth is a clojure salesforce library. It is specified with clojure.spec.alpha.
```

Comment:
That might have been manual, but I don't even know anymore. What the hell _is_ the name of this spec thing.